db/migrate/20161117153726_add_user_password_cost.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserPasswordCost < ActiveRecord::Migration
+  def change
+    add_column :users, :password_cost, :string
```

Comment:
I don't think there's anything to backfill. If the `password_cost` is nil it will default to what is in the code (hence all the comments above about how `COST` vs `cost`).